public/app/types/plugins.ts

```diff
@@ -17,3 +17,35 @@ export interface PluginMetaInfo {
     small: string;
   };
 }
+
+export interface PluginInfo {
```

Comment:
this is the same as the partial PluginMetaInfo type above. Think you can add these to that type and use that